buildlib/SSEFlags.cmake

```diff
@@ -0,0 +1,26 @@
+# check for SSE extensions
+include(CheckCSourceRuns)
+set(SSE_FLAGS)
+
+set(CMAKE_REQUIRED_FLAGS "-msse")
```

Comment:
The standard test protocol is to run the test in the current environment and look for failure.

If there is failure, then try and fix it with some compiler change.

So the above shouldn't be done, and the test should be changed to only check xmm it on i386, otherwise always pass.